pkg/daemon/ceph/osd/daemon.go

```diff
@@ -220,6 +220,15 @@ func getAvailableDevices(context *clusterd.Context, desiredDevices []DesiredDevi
 				} else if device.Name == desiredDevice.Name {
 					logger.Infof("%s found in the desired devices", device.Name)
 					matched = true
+				} else if desiredDevice.Name[0] == '/' {
```

Comment:
- nit: you could use `strings.HasPrefix()`
- Shall we check for the `/dev` prefix instead of only `/`?